sql/src/test/java/io/crate/metadata/ReferenceTest.java

```diff
@@ -67,4 +89,33 @@ public void testStreaming() throws Exception {
 
         assertThat(reference2, is(reference));
     }
+
+    @Test
+    public void testStreamingWithDefaultExpression() throws Exception {
+        ReferenceIdent referenceIdent = new ReferenceIdent(tableInfo.ident(), "int_column");
+        ExpressionAnalyzer analyzer
+            = new ExpressionAnalyzer(executor.functions(),
```

Comment:
the analyzer is unused now, removing it make `executor` unused.
I'd prefer to build the `RelationName` by hand here instead of creating a table definition using the executor as this is expensive and not really needed.